components/ide/code/extensionhelper/main.go

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"sync"
+	"time"
+
+	yaml "gopkg.in/yaml.v2"
+
+	gitpod "github.com/gitpod-io/gitpod/gitpod-protocol"
+)
+
+const (
+	GITPOD_REPO_ROOT             = "GITPOD_REPO_ROOT"
```

Comment:
it is better to use supervisor info endpoint, we should not rely on env vars, they are subject to change

I guess it was debt in VS Code Server code